db/data_migration/20160512150912_republish_corporate_information_pages.rb

```diff
@@ -0,0 +1,7 @@
+CorporateInformationPage
+  .published
+  .includes(:document)
+  .(:document, :translations)
```

Comment:
It might've been a typo, but it is valid ruby.  If the receiver of the message is a `Proc` then the `.()` message is short-hand for `call`.  E.g.

```ruby
proc = ->() { 1 }

proc.call() # => 1
# is equivalent to 
proc.() # => 1
```